_i18n/en.yml

```diff
@@ -142,8 +142,7 @@ developers_page:
       p_1: Follow ongoing feature development, user experience improvement, and security reviews in our [open source repository](https://github.com/18F/identity-idp){:target="_blank"}.
   footer:
     heading: Contact us
-    p_1: If you’re interested in learning more about the platform, please
-    a_1: contact us
+    p_1: If you’re interested in learning more about the platform, please email [partners@login.gov](mailto:partners@login.gov).
```

Comment:
Do we have people checking this email address yet? If not, I would say continue to use the login-contact@gsa.gov one from above